plugins/inputs/statsd/statsd.go

```diff
@@ -763,6 +764,10 @@ func (s *Statsd) parseName(bucket string) (name string, field string, tags map[s
 	}
 
 	name = bucketparts[0]
+	name = strings.ReplaceAll(name, " ", "_")
+	name = strings.ReplaceAll(name, "/", "-")
+	allowedChars := regexp.MustCompile(`[^a-zA-Z_\-0-9\.;=]`)
+	name = allowedChars.ReplaceAllString(name, "")
```

Comment:
After talking with @sspaink consider making this an option and use a table of regex + replacement
See regex from [this issue](https://github.com/statsd/statsd/issues/110)